.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    reviewers:
+      - Nikita-Smirnov-Exactpro
+      - OptimumCode
+    labels:
+      - dependencies
```

Comment:
issue: please, add this label to the repository

Reply:
Thank you for reminder